src/frontends/pytorch/src/op/transpose.cpp

```diff
@@ -81,7 +86,50 @@ OutputVector translate_t(const NodeContext& context) {
         if_node->set_input(input, param_then, param_else);
         return {if_node->set_output(result_then, result_else)};
     }
-}
+};
+
+OutputVector translate_movedim(const NodeContext& context) {
+    // aten::movedim.int(Tensor(a) self, int source, int destination) -> Tensor(a)
+    // aten::movedim.intlist(Tensor(a) self, int[] source, int[] destination) -> Tensor(a)
+    // based on https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/TensorShape.cpp#L3816
+    num_inputs_check(context, 3, 3);
+    auto x = context.get_input(0);
+    auto src_dims = context.get_input(1);
+    auto dst_dims = context.get_input(2);
+    Output<Node> rank;
+    std::tie(std::ignore, rank) = get_shape_rank(context, context.get_input(0), true);
+    src_dims = normalize_axis(context, src_dims, rank);
+    dst_dims = normalize_axis(context, dst_dims, rank);
+    auto start = context.mark_node(v0::Constant::create(element::i32, {}, {0}));
+    auto step = context.mark_node(v0::Constant::create(element::i32, {}, {1}));
+    auto range = context.mark_node(std::make_shared<v4::Range>(start, rank, step, element::i32));
+    auto range_src = context.mark_node(std::make_shared<v4::Range>(start, rank, step, element::i32));
+    auto range_dst = context.mark_node(std::make_shared<v4::Range>(start, rank, step, element::i32));
+    auto dims_1d_shape = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {-1}));
+    // operation accepts 0d and 1d source and destination, make them always 1d
+    src_dims = context.mark_node(std::make_shared<v1::Reshape>(src_dims, dims_1d_shape, false));
+    dst_dims = context.mark_node(std::make_shared<v1::Reshape>(dst_dims, dims_1d_shape, false));
+    auto axis_0 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
```

Comment:
`start` and `axis_0` can be a single zero constant

Reply:
reused